frontend/server/src/DAO/Tags.php

```diff
@@ -75,4 +75,46 @@ public static function findPublicTagsByPrefix(
 
         return $results;
     }
+
+    /**
+     * @return list<array{alias: string}>
+     */
+    public static function getFrequentsByLevel(
```

Comment:
```suggestion
    public static function getFrequentTagsByLevel(
```

normalmente debe haber un sustantivo en los nombres de las funciones. la función anterior (`findPublicTagsByPrefix`) es un ejemplo.